test/cppsuite/tests/basic_test.cxx

```diff
@@ -0,0 +1,157 @@
+/*-
+ * Public Domain 2014-present MongoDB, Inc.
```

Comment:
Lets rename this to something like "csuite_style_test.cxx" and leave a comment at the top that explains what it is and why its there, I don't want people thinking this is the way you're supposed to use the framework.

Reply:
See 4ca5a96ab836dedad68b07555415ad14e38ea251